lacework/provider.go

```diff
@@ -54,6 +54,7 @@ func Provider() terraform.ResourceProvider {
 			"lacework_alert_channel_gcp_pub_sub":     resourceLaceworkAlertChannelGcpPubSub(),
 			"lacework_alert_channel_jira_cloud":      resourceLaceworkAlertChannelJiraCloud(),
 			"lacework_alert_channel_jira_server":     resourceLaceworkAlertChannelJiraServer(),
+			"lacework_alert_channel_new_relic":       resourceLaceworkAlertChannelNewRelic(),
```

Comment:
uughh, I changed my mind.... Look at this:

https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/alert_channel

Should we rename it to `lacework_alert_channel_newrelic`? I would say so.... 
```suggestion
			"lacework_alert_channel_newrelic":       resourceLaceworkAlertChannelNewRelic(),
```